maintainers/maintainer-list.nix

```diff
@@ -8170,6 +8170,12 @@
     githubId = 483735;
     name = "Dmitry Geurkov";
   };
+  truh = {
```

Comment:
IIRC, adding yourself as a maintainer is preferred to be a separate commit.
Something like, "maintainers: add truh". Then apply the plantuml-server commit.

Reply:
done